### DISCUSSION TITLE
Operation not permitted FileException

### DISCUSSION THREAD

**Discussion by EdwarElrik** [abraunegg/onedrive#2954] [Q&A] [answered]
Hi all, can you please give me some light regarding this issue I am getting when running the sync:
```
std.file.FileException@std/file.d(1575): Forte PC/OLD/()Filename): Operation not permitted
----------------
??:? @trusted void std.file.setTimesImpl(scope const(char)[], scope const(char)*, std.datetime.systime.SysTime, std.datetime.systime.SysTime) [0x7f3245b015c8]
??:? [0x558d490ab597]
??:? [0x558d490a62f7]
??:? [0x558d490a897d]
??:? [0x558d490a2795]
??:? [0x558d49094be0]
??:? [0x558d49091ea8]
??:? [0x558d4914f6c1]
??:? void rt.dmain2._d_run_main2(char[][], ulong, extern (C) int function(char[][])*).runAll() [0x7f324579db7b]
??:? _d_run_main2 [0x7f324579d995]
??:? _d_run_main [0x7f324579d7ed]
??:? [0x558d491837a1]
??:? [0x7f324541c249]
??:? __libc_start_main [0x7f324541c304]
??:? [0x558d49057220]
Please wait, starting service ...
```

**Comment by abraunegg** [comment]
@EdwarElrik 
First lets start with a basic question - what application version are you running? What is the output of `onedrive --version` ?

**Comment by EdwarElrik** [comment]
onedrive v2.5.2-1+np1+1.1
Is running on OMV
It stopped working around a month ago.

**Comment by abraunegg** [comment, marked as answer]
@EdwarElrik 
> onedrive v2.5.2-1+np1+1.1

Please raise a bug report and generate a debug log for support following this process: https://github.com/abraunegg/onedrive/wiki/Generate-debug-log-for-support

If you are unsure how to do this, raise a question on the OMV forums on how to do this correctly on that platform.

> It stopped working around a month ago.

It will most likely be permission related, but the application is not handling the permission issue. This is why the debug log is needed to see where the application is breaking.
